src/reducers/reducer.js

```diff
@@ -0,0 +1,8 @@
+import {combineReducers} from 'redux';
+import filter from './filter';
+import allCourses from './course';
+
+export default combineReducers({
+  allCourses,
+  filter
+});
```

Comment:
Hva tenker du: Er det bedre å ha reducer.js under src/ istedet for src/reducers?